drizzlepac/outputimage.py

```diff
@@ -530,32 +538,37 @@ def writeFITS(self, template, sciarr, whtarr, ctxarr=None,
                 # header to Primary header...
                 if dqhdr:
                     for _card in dqhdr.cards:
-                        if ( (_card.keyword not in RESERVED_KEYS) and
+                        if ((_card.keyword not in RESERVED_KEYS) and
                              _card.keyword not in hdu.header):
                             hdu.header.append(_card)
                 hdu.header['filename'] = self.outcontext
+
                 if self.wcs:
-                    pre_wcs_kw = self.find_kwupdate_location(hdu.header,'CD1_1')
+                    pre_wcs_kw = self.find_kwupdate_location(hdu.header, 'CD1_1')
                     # Update WCS Keywords based on PyDrizzle product's value
                     # since 'drizzle' itself doesn't update that keyword.
-                    addWCSKeywords(self.wcs,hdu.header, blot=self.blot,
+                    addWCSKeywords(self.wcs, hdu.header, blot=self.blot,
                                    single=self.single, after=pre_wcs_kw)
 
+                # explicitly set EXTNAME, EXTVER to (sci,1)
+                hdu.header.set('EXTNAME', value='CTX', before='TELESCOP')
```

Comment:
Comment should be updated.

Reply:
Although this code has been replaced, the new keyword being added does include a comment as implemented in 89fa478.